pkg/gatewayconfig/shared/shared_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
+
+import (
+	"os"
+	"testing"
+
+	"github.com/smartystreets/assertions"
+	"github.com/smartystreets/assertions/should"
+	"go.thethings.network/lorawan-stack/pkg/fetch"
+	"go.thethings.network/lorawan-stack/pkg/frequencyplans"
+)
+
+func getFetcher(fsPath, remotePath string) fetch.Interface {
+	var fetcher fetch.Interface
+	localPath := os.ExpandEnv(fsPath)
+	if _, err := os.Stat(localPath); err == nil {
+		fetcher = fetch.FromFilesystem(localPath)
+	} else {
+		fetcher = fetch.FromHTTP(remotePath, true)
+	}
+	return fetcher
+}
+
+func TestSX1301Conf(t *testing.T) {
+
+	freqPlanFetcher := getFetcher("$GOPATH/src/github.com/TheThingsNetwork/lorawan-frequency-plans", "https://raw.githubusercontent.com/TheThingsNetwork/lorawan-frequency-plans/master")
```

Comment:
Remove `$GOPATH` dependencies; with modules we're not checked out there necessarily. Use `testdata` or relative path instead

Reply:
Also respect an offline test option as we do elsewhere

Reply:
I've removed the online test that I introduced for the `shared` package and used a hardcoded test data since the `semtechudp` package covers this.
The original tests (in `semtechudp`) by @htdvisser do not have an offline option.
If this change has to be made, could you please point out some examples in the code base which I can adapt from?